### DISCUSSION TITLE
Why can't I download the extension from edge store?

### DISCUSSION THREAD

**Discussion by rahaaatul** [uBlockOrigin/uBOL-home#246] [Q&A] [answered]
I get this error:

```
An error has occurred
There was a problem getting the manifest file for this add-on. Please try again.
```

![demo-video](https://github.com/user-attachments/assets/6faa2e1c-0146-400a-8800-4316c75de063)

What's wrong and how do I fix this?

**Comment by gorhill** [comment, marked as answer]
No clue. Apparently this also happens for uBlock Origin. Best is to ask Microsoft Edge support.
